joss-paper/paper.md

```diff
@@ -0,0 +1,250 @@
+---
+title: 'Singularity-EOS: Performance Portable Equations of State and Mixed Cell Closures'
+tags:
+  - C++
+  - Fortran
+  - Python
+  - equations of state
+  - thermodynamics
+  - performance portability
+authors:
+  - name: Jonah M. Miller
+    orcid: 0000-0001-6432-7860
+    affiliation: "1, 2"
+    corresponding: true
+  - name: Daniel A. Holladay
+  - orcid: 0000-0002-0673-9741
+    affiliation: "2, 3"
+  - name: Jeffrey H. Peterson
+    orcid: 0000-0001-9425-4674
+    affiliation: 4
+  - name: Christopher M. Mauney
+    affiliation: "2, 5"
+  - name: Richard Berger
+    orcid: 0000-0002-3044-8266
+    affiliation: 3
+  - name: Anna Pietarila Graham
+    affiliation: 5
+  - name: Karen Tsai
+    affiliation: 3
+  - name: Brandon Barker
+    affiliation: "1, 2, 6, 7"
+  - name: Alexander Holas
+    affiliation: "2, 3, 8"
+  - name: Ann E. Wills
```

Comment:
My publishing name is Ann E. Mattsson, but you could use Ann E. Mattsson Wills if needed.

Reply:
I will change to Ann E. Mattsson. Was forced to change to Ann E. Wills during RASSTI.

Reply:
Fixed.